data/output/UAT_direct/translations_simple.csv

```diff
@@ -0,0 +1,304 @@
+title_id;title_nl;title_fr;title_en
+first_observed;jaar van eerste waarneming;année de première observation;year of first observation
+last_observed;jaar van laatste waarneming;année de dernière observation;year of last observation
+kingdom;rijk;règne;kingdom
+phylum;stam;phylum;phylum
+class;klasse;classe;class
+order;orde;ordre;order
+family;familie;famille;family
+locality;locatie;localité;locality
+locationId;locatie-ID;ID de la localité;location ID
+native_range;natuurlijk areaal;aire d'origine;native range
+degree_of_establishment;mate van vestiging;degré d'établissement;degree of establishment
+pathway;introductieroute;voie d'introduction;pathway
+pathway_level1;niveau 1;niveau 1;level 1
+pathway_level2;niveau 2;niveau 2;level 2
+habitat;habitat;habitat;habitat
+marine;marien;marine;marine
+freshwater;zoetwater;eau douce;freshwater
+terrestrial;terrestrisch;terrestre;terrestrial
+source;Bron;Source;Source
+species;Soort;Espèce;Species
+native_continent;continent van oorsprong;continent d'origine;native continent
+more;Meer;Plus;More
+less;Minder;Moins;Less
+gbifLink;GBIF-sleutel (taxonkey);Clé GBIF (taxonkey);GBIF taxonkey
```

Comment:
Ik ben niet verkocht voor sleutel ! maar ik weet niet wat we best wel pakken.

Reply:
Wat dacht je van GBIF soort-ID of GBIF ID?

Reply:
Beter 👍

Reply:
Heb er GBIF taxonomisch ID van gemaakt :)

Reply:
Nog beter 😄